### DISCUSSION TITLE
V3 - Port 5000 - bad gatway - web

### DISCUSSION THREAD

**Discussion by leoborlot** [postalserver/postal#2969] [Development discussions]
Hello everyone,

I appreciate all the work you do with Postal.

I'm facing a challenge. I use Traefik instead of Caddy as a reverse proxy, and I don't run the application in Host mode. Instead, I prefer using 'docker networks' because I need other containers to communicate at the same level, and for practicality, I prefer to change Postal's network. I have a good understanding of Docker and have adjusted Postal's docker-compose file to work as I need.

I performed this same operation on Postal version 2, and everything is running correctly.

However, now I'm planning to upgrade to Postal v3 and did a fresh installation, following the same logic within Docker. I've encountered a technical difficulty. When I take the web container out of Host mode, it doesn't allow me to access port 5000.

Please see the attached image.

![curl postal](https://github.com/postalserver/postal/assets/20361821/f1a6c2f4-b9b3-4501-85f2-07a8cdf66a18)

1 - The first request is from within the container. If I access port 5000 via localhost, it accepts the connection, although there is no 'response'.

2 - But if I access the port using the container's IP in the network, it gives a 'Connection Refused' response.

### If I make the same request in version 2, it responds correctly in both ways.

Any on know how may I be this por as free access from outside?

**Comment by willpower232** [comment]
Are you able to share your `docker ps -a` to see all docker containers and their ports?
